### ISSUE TITLE
Add onPress() event to Component

### ISSUE THREAD

**Issue by emmanuel-D** [ihmpavel/expo-video-player#261]
In order to get more control on the VideoPlayer Component,  I would like to handle the action when an User presses on the Component (the video itself without considering the controls) via an **event** like `onPress()`.

**Comment by soheils2** [comment]
after hours of exploring i found this that may help you/::

onStartShouldSetResponder() :
thisfuction calls evrytime you click on player without making problem with native controllers or any component you want to show on screen with fixed position

good new is it has touch event for farther usage
`
<VideoPlayer
            videoProps={{
              ref: component => this._Player = component,
              source: {
                uri: this.state.currentUri
              },
              onStartShouldSetResponder: (evt) => { 
              console.log("yyayyy:", Math.floor(evt.nativeEvent.locationX), Math.floor(evt.nativeEvent.locationY));
             },
            }}
            hideControlsTimerDuration={5000}
            playbackCallback={(stt) => { this.setState({ status: stt }); }}
          
          />
`

**Comment by emmanuel-D** [comment]
Thanks for your answer
I tested it and it works as expected for my use case.

I also noticed that the video is automatically loading data when the `VideoPlayer` is rendered, despite the fact that I always set the props `shoulPlay` by default to `false`.
And this lead to over mobile data consumption because, there is also no caching possibly.

So is there a way to set a thumbnail image and only load video data when the User presses on the `VideoPlayer` ?
